### DISCUSSION TITLE
Is it time for this to become a fork?

### DISCUSSION THREAD

**Discussion by shellscape** [VSCodium/vscodium#1357] [General]
I fully understand that vscodium was not intended to be a fork of vscode. I also understand that this project attempts to stay in line with upstream for updates and changes. I spent a decent amount of time trying to contribute to the upstream project, only to run into Microsoft employees who shut down the mere discussion of enhancements. Upstream is being managed and gatekept exclusively by Microsoft-aligned people, and there's no room for improvement outside of internal product sprints. That begs the question:

Is it time for this project to become a fork for which the community can submit enhancements to?

Should we start a fork under the vscodium github org, and attempt to keep that fork inline with vscodium?

**Comment by daiyam** [comment]
This is exactly how I've got involved with VSCodium, I do use it for [my fork](https://github.com/zokugun/MrCode) (not up to date, will be soon)...
I'm conflicted but I think we should keep VSCodium with the fewest changes and have an easy the process to make forks.

> Should we start a fork under the vscodium github org, and attempt to keep that fork inline with vscodium?

We could have of "fork" of vscodium (like vscode/vscodium) under the vscodium org.
